.github/CONTRIBUTING.md

```diff
@@ -1,32 +1,44 @@
-
-# Development
+# Contributing to SimpleITK
 
 We welcome contributions from the community!
 
-SimpleITK uses the [Git](http://git-scm.com/) distributed version control system. The main repository is hosted on [itk.org](https://itk.org/SimpleITK.git) and mirrored to [Github](https://github.com/SimpleITK/SimpleITK.git). We are currently using Github for pull requests, and issue tracking.
+SimpleITK uses the [Git](http://git-scm.com/) distributed version control system. The main repository is hosted on [Github](https://github.com/SimpleITK/SimpleITK.git) and mirred on [itk.org](https://itk.org/SimpleITK.git) and mirrored on the [Insight Software Consortium](https://github.com/InsightSoftwareConsortium/SimpleITK). For pull requests and issue tracking  Github is also used.
```

Comment:
The main repository is hosted on [Github](https://github.com/SimpleITK/SimpleITK.git) and mirred on [itk.org](https://itk.org/SimpleITK.git) and mirrored on the [Insight Software Consortium](https://github.com/InsightSoftwareConsortium/SimpleITK). 

Change to:

The main repository is hosted on [Github](https://github.com/SimpleITK/SimpleITK.git) and mirrored on [itk.org](https://itk.org/SimpleITK.git) and on the [Insight Software Consortium](https://github.com/InsightSoftwareConsortium/SimpleITK).